static/sass/styles.scss

```diff
@@ -163,3 +163,12 @@
 .u-row-selected {
   background-color: $color-mid-x-light;
 }
+
+.u-instance-status {
```

Comment:
better use the vanilla standard `u-text--muted` `p-text--small`. We should avoid custom CSS.

Reply:
Done @ [4f62c9e](https://github.com/canonical/lxd-ui/commit/4f62c9e2df86d2b692c4298deac8212f25ffcc02)

(`u-text--muted` not needed here)